Test/Float/FloatArithmetic.pml

```diff
@@ -19,15 +19,15 @@ active proctype test() {
     float rdiv_lower;
     makeFloat(rdiv_lower, 74, 100);
     makeFloat(rdiv_upper, 76, 100);
-    a = 12;
```

Comment:
I think we could remove makeFloat now, right?

Reply:
If you wish...